Marlin/pins.h

```diff
@@ -230,6 +230,9 @@
 #ifndef HEATER_3_PIN
   #define HEATER_3_PIN -1
 #endif
+#ifndef HEATER_4_PIN
+  #define HEATER_4_PIN -1
+#endif
```

Comment:
Well, I might be wrong, but now Marlin supports "fifth" extruder and heater? (E0 ... E4, H0 ... H4)